drivers/ethernet/eth_lan865x.c

```diff
@@ -66,47 +66,46 @@ static int lan865x_mac_rxtx_control(const struct device *dev, bool en)
 	return oa_tc6_reg_write(ctx->tc6, LAN865x_MAC_NCR, ctl);
 }
 
-static int lan865x_enable_sync(const struct device *dev)
+static void lan865x_iface_init(struct net_if *iface)
 {
+	const struct device *dev = net_if_get_device(iface);
 	struct lan865x_data *ctx = dev->data;
-	uint32_t val;
+	struct oa_tc6 *tc6 = ctx->tc6;
 	int ret;
 
-	ret = oa_tc6_reg_read(ctx->tc6, OA_CONFIG0, &val);
+	ret = oa_tc6_enable_sync(tc6);
 	if (ret) {
-		return ret;
+		LOG_ERR("Sync enable failed: %d\n", ret);
+		return;
 	}
-	val |= OA_CONFIG0_SYNC | OA_CONFIG0_RFA_ZARFE;
-	ret = oa_tc6_reg_write(ctx->tc6, OA_CONFIG0, val);
+
+	ret = oa_tc6_zero_align_receive_frame_enable(tc6);
 	if (ret) {
-		return ret;
+		LOG_ERR("ZARFE enable failed: %d\n", ret);
```

Comment:
is the return ret missing or should this only be a warning? can this be recovered?